TUM-Dev/NavigaTUM
# Add Roomfinder-like maps for CH

## Proposed Changes (include Screenshots if possible)

- Add Roomfinder map  images for our current chemie overlays (mainly so that they can be directly seen in the TCA)
- This is preliminary until we have proper data (see #403)

![image](https://user-images.githubusercontent.com/7429408/220980827-8484ffed-0b50-47bb-bebd-9d6fcf8e7ab7.png)


## How to test this PR

1. Recompile the latest data
2. Verify that the map images are available in your local webserver
3. Open rooms where we have coordinates (e.g. 5402.01.221K)

## How has this been tested?

- Multiple rooms in different floors

## Checklist:
